cardiopy/ekg.py

```diff
@@ -765,19 +765,17 @@ def calc_psd_welch(self, itype, window):
 
 
     def calc_psd_mt(self, bandwidth):
-        """ Calculate multitaper power spectrum 
-
-            Params
-            ------
-            bandwidth: float
-                frequency resolution (NW)
+        """
+        Calculate multitaper power spectrum.
 
-            Returns
```

Comment:
Can this be switched to the "modifies" section so that it's clear that the EKG.psd_mt attribute is created?

Reply:
New push should resolve this

Reply:
thanks!